### ISSUE TITLE
Order Scout Reports by visit date on Archaeological Resource page

### ISSUE THREAD

**Issue by kkemp85** [legiongis/hmsf-md#233]
I know we had a ticket about this a while back and we got it fixed up, but when we did the last big upgrade I think it reverted back. We would like the Scout Reports to be listed from oldest to newest visit date on each resource's page. It's hard to tell older reports from new ones and track change through time once you get a lot of reports logged.

<img width="293" alt="image" src="https://user-images.githubusercontent.com/54329725/150232344-48ce6737-0c9a-4ee7-8805-b9640518b3e4.png">

**Comment by mradamcox** [comment]
Good catch, I believe this happened because the update brought us back to the default Arches way for acquiring this list of related resources. I'll look into how we can get this ordered.

**Comment by mradamcox** [comment]
Looks like this got handled a few months ago here: https://github.com/legiongis/hmsf-md/commit/7d42ac54f0ffdc22993a078dc2d1ad782f41f207#diff-bf31bc9a5de90408e90422508692cbd49038b164690df4349ba9a33a8b8c8e5c

<img width="2126" height="1569" alt="Image" src="https://github.com/user-attachments/assets/8e62679b-a23f-48d3-842f-9c18e1575298" />
